src/dropin.js

```diff
@@ -97,6 +97,8 @@ HAS_RAW_PAYMENT_DATA[constants.paymentMethodTypes.applePay] = true;
  * @property {string} details.cardType Type of card, ex: Visa, Mastercard.
  * @property {string} details.lastFour The last 4 digits of the card.
  * @property {string} details.lastTwo The last 2 digits of the card.
+ * @property {boolean} details.isNetworkTokenized True if the card is network tokenized.
```

Comment:
```suggestion
 * @property {boolean} details.isNetworkTokenized True if the card is network tokenized. A network tokenized card is a generated virtual card with a device-specific account number (DPAN) that is used in place of the underlying source card.
```

Reply:
Pulled this description from our [reference docs](https://developers.braintreepayments.com/reference/response/android-pay-card/java#is_network_tokenized), this is a term that gets frequent inbounds from confused merchants.

Reply:
Should we be even more explicit and say that 3ds will run against these cards automatically (if 3ds is enabeld) if they are not network tokenized?